### PR TITLE
Add CI testing on a ppc64le OpenStack cluster at OSU's Open Source Lab

### PR DESCRIPTION
**Pull request by hamzy**
This is a cluster of ppc64le machines.

The major difference is that the floating ips are predefined.

### CODE REVIEW

File: ci-operator/config/openshift/release/openshift-release-master__ocp-4.5-ci.yaml [hunk 1]
```diff
@@ -19,6 +19,21 @@ tests:
...
     workflow: openshift-upgrade-aws
+- as: e2e-openstack-osuosl-ipi
+  cron: 0 8,20 * * *
+  steps:
+    cluster_profile: openstack-osuosl
+    test:
+    - as: e2e
+      commands: |-
+        #!/bin/bash
+        e2e-tests # as built by go test -c
```
Comment (droslean) [inline ci-operator/config/openshift/release/openshift-release-master__ocp-4.5-ci.yaml:30]: Is it needed to put this command in a bash script?

Comment (hamzy) [reply ci-operator/config/openshift/release/openshift-release-master__ocp-4.5-ci.yaml:30]: Good question @droslean .  Maybe @stevekuznetsov knows definitively.  I took this example from elsewhere.

**Comment by hamzy** [comment]
retest

File: ci-operator/config/openshift/release/openshift-release-master__ocp-4.5-ci.yaml (hunk 1 above)
Comment (stevekuznetsov) [reply ci-operator/config/openshift/release/openshift-release-master__ocp-4.5-ci.yaml:42]: No, there's no need. Better left without

File: ci-operator/config/openshift/release/openshift-release-master__ocp-4.5-ci.yaml [hunk 2]
```diff
@@ -8,6 +13,11 @@ releases:
       product: ocp
       stream: ci
       version: "4.5"
+  release-ppc64le:
```
Comment (stevekuznetsov) [inline ci-operator/config/openshift/release/openshift-release-master__ocp-4.5-ci.yaml:16]: perfer `ppc64le` as the name

**Review by stevekuznetsov — COMMENTED** [review]
Importing the Power release and using it won't work unless you have Power infrastructure to run all of it on. For instance - your workflow will need the `cli` and `installer` images to run, but they'll run on the x86 hosts we have for infra.

**Pull request closed without merging**